### ISSUE TITLE
Megadrive gamepad support for all butttons, not only 2

### ISSUE THREAD

**Issue by Moyone** [ZXMicroJack/mist-firmware-rp2040#16]
It would be great to have  support for Megadrive gamepad protocol in order to be able to use all buttons in cores and menu.

**Comment by delgrom** [comment]
Yes, it would be great. I have seen support for MD pads in mist firmware for the microcontroller of Sidi128, maybe it helps you. Route is hw/ATSAMV71/hardware.c

I belive there is an option in mist.ini to enable/disable the support DB9 MD joysticks.
